src/main/resources/scripts/sequence-files/purge_sequence_files.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+import argparse
+import mysql.connector
+import os
+
+def remove(path, purge):
+    if purge:
+        try:
+            if os.path.exists(path):
+                if os.path.isdir(path):
+                    os.rmdir(path)
+                elif os.path.isfile(path):
+                    os.remove(path)
+                else:
```

Comment:
This else might have the wrong indentation? I don't think it can be reached and it was intended to be fore the if exists line.

Reply:
I would probably be a good idea to add a print statement when they get removed too, because it just says "All done." when it removes files, so if you mistype your directory and run it, we have no idea if the files were deleted or not before manually checking (this happened to me while testing it)

Reply:
Please see e77dc42b0bcf2e25ac53a3ebfb4e9ffa905ecf20.